nixos/modules/config/ipa.nix

```diff
@@ -0,0 +1,241 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.ipa;
+  sssd = config.services.sssd;
+  ntp = config.services.ntp;
+  krb5 = config.krb5;
+  ldap = config.users.ldap;
```

Comment:
```suggestion
```
We shouldn't create global aliases for them. Its confusing and not really standard. If required we can create them more locally.